src/Orc.NuGetExplorer.Xaml/Extensions/IBusyIndicatorServiceExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Orc.NuGetExplorer
+{
+    using System;
+    using Catel;
+    using Catel.Services;
+
+    public static class IBusyIndicatorServiceExtensions
+    {
+        public static IDisposable WaitingScope(this IBusyIndicatorService busyIndicatorService)
```

Comment:
Don't we have PushInScope already? Or is it in ORchestra? We might have to move it to Orc.Theming